src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,154 @@
+package duke;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Storage {
+
+    // The relative path to the directory
+    private final String directory;
+
+    // The file
+    private final String file;
+
+    // The contents of the file as a List of Strings
+    private ArrayList<String> fileTasks;
+
+    /**
+     * Constructor
```

Comment:
Perhaps need a full stop at the end? 🤔
And I noticed the same issue in several other places below.